primo/data_parser/default_data.py

```diff
@@ -0,0 +1,260 @@
+#################################################################################
+# PRIMO - The P&A Project Optimizer was produced under the Methane Emissions
+# Reduction Program (MERP) and National Energy Technology Laboratory's (NETL)
+# National Emissions Reduction Initiative (NEMRI).
+#
+# NOTICE. This Software was developed under funding from the U.S. Government
+# and the U.S. Government consequently retains certain rights. As such, the
+# U.S. Government has been granted for itself and others acting on its behalf
+# a paid-up, nonexclusive, irrevocable, worldwide license in the Software to
+# reproduce, distribute copies to the public, prepare derivative works, and
+# perform publicly and display publicly, and to permit others to do so.
+#################################################################################
+
+# Standard libs
+from dataclasses import dataclass
+from typing import Union
+
+# This file should contain all the constants
+FEASIBILITY_TOLERANCE = 1e-6  # Optimization Feasibility tolerance
```

Comment:
@radhakrishnatg : These are currently duplicated in primo\utils\__init__.py and primo\opt_model\__init__.py. Could you please update the code to remove duplication ?

Reply:
There are other files/notebooks which are importing these values from `utils/__init__.py`. So, instead of duplicating the values in the __init__ file,  I have imported them from the default_data.py file. See `primo/utils/__init__.py` file for an example. This way, there is no duplication, and at the time same time all the remaining files/notebooks work normally.